code/game/mecha/combat/gorilla.dm

```diff
@@ -0,0 +1,219 @@
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/cannon
+	name = "8.8cm KwK 47"
+	desc = "Do to enemy mechs what the King Tiger did to Allied tanks with 88 milimeters of armor-piercing German steel!"
+	icon_state = "mecha_uac2"
+	equip_cooldown = 55 // 5.5 seconds
+	projectile = /obj/item/projectile/bullet/cannon
+	fire_sound = 'sound/weapons/cannon.ogg'
+	projectiles = 1
+	projectile_energy_cost = 1000
+	salvageable = 0 // We don't want players ripping this off a dead mech.
+
+/obj/item/projectile/bullet/cannon
+	name ="armor-piercing shell"
+	icon = 'icons/obj/grenade.dmi'
+	icon_state = "shell"
+	damage = 1000 // In order to 1-hit any other mech and royally fuck anyone unfortunate enough to get in the way.
+
+	on_hit(var/atom/target, var/blocked = 0)
+		explosion(target, 0, 0, 2, 4)
+		return 1
+
+/* // GLITCHY UND LAGGY.
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/mg42
+	name = "Maschinengewehr 60"
+	icon_state = "mecha_uac2"
+	equip_cooldown = 10
+	projectile = /obj/item/projectile/bullet/midbullet2
+	fire_sound = 'sound/weapons/mg42.ogg'
+	projectiles = 1000
+	projectiles_per_shot = 5
+	deviation = 0.3
+	projectile_energy_cost = 20
+	fire_cooldown = 1
+	salvageable = 0 // We don't want players ripping this off a dead mech.
+*/
+
+/obj/effect/decal/mecha_wreckage/gorilla
+	name = "Gorilla wreckage"
+	desc = "... Blitzkrieg?"
+	icon = 'icons/mecha/mecha64x64.dmi'
+	icon_state = "pzrwreck"
+	layer = 4 // so it overlaps other people
+	pixel_x = -16
+	anchored = 1 // It's fucking huge. You aren't moving it.
```

Comment:
Is there a way to get rid of this wreckage?

Reply:
Dismantle it.

Reply:
Ah, alright. Thanks.